src/Sylius/Bundle/ApiBundle/spec/CommandHandler/Checkout/CompleteOrderHandlerSpec.php

```diff
@@ -52,7 +55,11 @@ function it_handles_order_completion_without_notes(
 
         $stateMachine->apply('complete')->shouldBeCalled();
 
-        $emailManager->sendConfirmationEmail($order)->shouldBeCalled();
+        $orderCompleted = new OrderCompleted('COMPLETED_ORDER_TOKEN');
+
+        $eventBus->dispatch($orderCompleted, [new DispatchAfterCurrentBusStamp()])
```

Comment:
I'm surprised, that it works. I would expect, that the second argument makes a problem with matching to the actual value.